AS_923.yml

```diff
@@ -0,0 +1,38 @@
+band-id: AS_923
+uplink-channels:
+- frequency: 923200000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+- frequency: 923400000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+downlink-channels:
+- frequency: 923200000
+  min-data-rate: 0
+  max-data-rate: 5
+- frequency: 923400000
+  min-data-rate: 0
+  max-data-rate: 5
+lora-standard-channel:
+  frequency: 922100000
+  data-rate: 6
+  radio: 1
+fsk-channel:
+  frequency: 921800000
+  data-rate: 7
+  radio: 1
```

Comment:
The high-bandwidth and FSK channels don't belong in here. Only the two mandatory channels.